README.md

```diff
@@ -522,6 +522,43 @@ Usable properties for an unsubscribe `question` type are `title`, `content`, `bu
 The default `data_method` is `delete`. The `data_method` should typically not be specified as it should correspond with the `unsubscribe_url` that is supplied by the system when calling the questionnaire generator. Only when we call this private function with `send` to show a card on the mentor dashboard is when we override both the `unsubscribe_url` and the `data_method` but it's a bit of a hack.
 
 
+
+### Type: Dropdown
+Required and allowed options (minimal example and maximal example):
+
+```ruby
+[{
+  id: :v1,
+  type: :dropdown,
+  title: 'Waar hadden de belangrijkste gebeurtenissen mee te maken?',
+  options: ['hobby/sport', 'werk', 'vriendschap', 'romantische relatie', 'thuis']
+}, {
+  section_start: 'De hoofddoelen',
+  hidden: true,
+  id: :v2,
+  type: :dropdown,
+  title: 'Aan welke doelen heb je deze week gewerkt tijdens de begeleiding van deze student?',
+  label: 'RMC regio',
+  tooltip: 'some tooltip',
+  options: ['hobby/sport', 'werk', 'vriendschap', 'romantische relatie', 'thuis'],
```

Comment:
Willen we hier ook nog shows / hides aan toevoegen? Of is dat niet nodig voor deze questionnaire?

Reply:
Dat is niet nodig en dat is volgens mij ook belachelijk moeilijk om te doen zolang we dit niet in react hebben omdat we dan voor hetzelfde element verschillende javascript code moet uitgevoerd worden op basis van wat is geselecteerd ipv gewoon aan een bepaalde checkbox of radio optie een event te binden.